WorkloadManagementSystem/Executor/JobScheduling.py

```diff
@@ -354,7 +354,9 @@ def __resolveStaging( self, jobState, inputData, idSites ):
     return ( True, bestSites )
 
   def __preRequestStaging( self, jobState, stageSite, opData ):
-    result = getSEsForSite( stageSite )
+    from DIRAC.DataManagementSystem.Utilities.DMSHelpers import DMSHelpers
```

Comment:
I would just reorganize the code differently. A bit bellow, you get the vo.
So instanciate the DMSHelpers after that, so that you can pass it the vo parameters. 
Also, the import can be done at the beginning of the file no ?